main.cpp

```diff
@@ -78,23 +76,21 @@ struct U
 
 struct UpdateThat
 {
-    static float updateNums(U* that, float* updated )        //10
+    static float updateNums(U& that, const float& updated )        //10
     {
-        if(that != nullptr && updated != nullptr)
+        //if(that != nullptr && updated != nullptr)
         {
```

Comment:
Don't just comment it out.  Delete the curly braces as well and fix the indentation
- [ ] revise